.gitignore

```diff
@@ -54,6 +55,9 @@ to_sync/
 src/projectM-sdl/build/
 src/libprojectM/build/
 *.pkg
+./fontend-qt
```

Comment:
Please remove this path.

```suggestion
```

For once, this path is misspelled. I'd also recommend not to put any arbitrary paths here which are highly user-specific. Reason is, every user has their own needs, and adding all those possibilities here would blow up the file a bit.

To elaborate further on this: building the Qt frontend (or any other application depending on libprojectM) should not be done this way around (e.g. adding the dependee inside its own dependency) - instead, either build libprojectM separately and use CMAKE_PREFIX_PATH to make the library install dir known to the frontent-qt project. Note that you currently can't just use `add_subdirectory` with libprojectM, as it's not set up to be built as a subproject. We'll probably change this in the future to make integrating libprojectM as a code dependency easier.

See our [Integration Quickstart Guide](https://github.com/projectM-visualizer/projectm/wiki/Integration-Quickstart-Guide) for best practices on using libprojectM in other applications.

Reply:
Good to know, thanks for the feedback and advice! Fixing now.